bin/esi-install-image

```diff
@@ -137,14 +159,23 @@ class ServiceImageManager(object):
         ### Bundle and upload image
         image_file = decompress_stdout.strip()
         print "Installing image from {0}".format(image_file)
-        version_string = self._get_image_version() if version is None \
-            else "0.{0}".format(version)
+        version_string = self._get_package_version() if version is None \
+            else "{0}".format(version)
         bucket = self.IMAGE_NAME + '-' + version_string
         image_name = "{0}-v{1}".format(self.IMAGE_NAME, version_string)
         print "Bundling, uploading and registering image to bucket: " + bucket
         install_cmd = "euca-install-image -b {0} -i {1} " \
-                      "-r x86_64 -n {2} --virt hvm".format(bucket, image_file,
-                                                           image_name)
+                      "-r x86_64 -n {2} " \
+                      "--virt hvm " \
+                      "--region {3} " \
+                      "-I {4} -S {5} " \
+                      "--user {6} --ec2cert {7}".format(
+                                            bucket, image_file,
+                                            image_name, self.region,
+                                            self.get_env_var('AWS_ACCESS_KEY_ID'),
+                                            self.get_env_var('AWS_SECRET_ACCESS_KEY'),
+                                            self.get_env_var('EC2_USER_ID'),
+                                            self.get_env_var('EUCALYPTUS_CERT'))
```

Comment:
See my previous commentary about shell injection and passing credentials with environment variables.